addons/im_livechat/controllers/chatbot.py

```diff
@@ -63,22 +62,34 @@ def chatbot_trigger_step(self, channel_id, chatbot_script_id=None):
             return None
 
         posted_message = next_step._process_step(discuss_channel)
-        return {
-            "data": Store(posted_message, for_current_user=True).get_result(),
-            'scriptStep': {
-                'id': next_step.id,
-                'answers': [{
-                    'id': answer.id,
-                    'label': answer.name,
-                    'redirectLink': answer.redirect_link,
-                } for answer in next_step.answer_ids],
-                'isLast': next_step._is_last_step(discuss_channel),
-                'message': plaintext2html(next_step.message) if not is_html_empty(next_step.message) else False,
-                'type': next_step.step_type,
+        store = Store()
+        store.add(posted_message, for_current_user=True)
+        store.add(next_step)
+        store.add(
+            "ChatbotStep",
+            {
+                "id": (next_step.id, discuss_channel.id),
+                "isLast": next_step._is_last_step(discuss_channel),
+                "message": store.one(posted_message, only_id=True),
+                "operatorFound": next_step.step_type == "forward_operator"
+                and len(discuss_channel.channel_member_ids) > 2,
+                "scriptStep": store.one(next_step, only_id=True),
             },
-            'operatorFound': next_step.step_type == 'forward_operator' and len(
-                discuss_channel.channel_member_ids) > 2,
-        }
+        )
+        store.add(
+            "Chatbot",
+            {
+                "currentStep": {
+                    "id": (next_step.id, discuss_channel.id),
+                    "scriptStep": next_step.id,
+                    "message": posted_message.id,
+                },
+                "id": (chatbot.id, discuss_channel.id, "discuss.channel"),
```

Comment:
"discuss.channel" is probably unnecessary as we're only using channels, and not actually inserting this value in thread